### PR TITLE
Modify UntagCommand to use tag instead of index

### LINKED ISSUE #54: Modify UntagCommand to use Tag instead of Index

### PR DESCRIPTION
**Pull request by li-s**
Resolve #54 
Milestone v1.2b
Milestone v1.2

### CODE REVIEW

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 1]
```diff
@@ -56,8 +56,9 @@ public void execute_invalidCommandFormat_throwsParseException() {
 
     @Test
     public void execute_commandExecutionError_throwsCommandException() {
-        String untagCommand = "untag 9";
-        assertCommandException(untagCommand, MESSAGE_INVALID_TAG_DISPLAYED_INDEX);
+        String tagname = "laksdjflkajsdlkfjlksadjl";
```
Comment (luo-git) [inline src/test/java/seedu/address/logic/LogicManagerTest.java]: Could this be changed to a more sensible tag name?

File: src/test/java/seedu/address/logic/commands/ModelStubDeleteTag.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            }
+        }
+        return new ArrayList<>();
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+
```
Comment (luo-git) [inline src/test/java/seedu/address/logic/commands/ModelStubDeleteTag.java]: Should there be empty lines here?

**Review by luo-git — CHANGES_REQUESTED** [review]
LGTM except some minor changes.

**Review by luo-git — APPROVED** [review]

**Review by anthony6401 — APPROVED** [review]
LGTM!

**Pull request merged**